cn-india/app.js

```diff
@@ -0,0 +1,37 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+const mongoose = require("mongoose");
+require("dotenv").config();
+const registerRoute=require('./routes/user')
+const isAuth=require('./middlewares/isAuth')
+const projectRouter=require('./routes/project')
+const dataRouter=require('./routes/addData')
+const publicRouter=require('./routes/public')
+const resetpassword=require('./routes/user')
+const verifyCD=require('./routes/cdverify')
```

Comment:
please rename this route to adminRoutes and also rename cdverify file to admin file